open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml

```diff
@@ -15,102 +15,106 @@
         <version>1.1-SNAPSHOT</version>
     </parent>
 
-    <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <derby.version>10.8.3.0</derby.version>
-    </properties>
+   <properties>
```

Comment:
I think the jacoco setup should be removed. We have jacoco defined at the top level, and the linux foundation build uses this for the sonar scan.

I did originally remove this in #595 -- did that break anything in the virtualizer?

Reply:
Hi, I've removed this part. This is the old version of virtualizer which will be deprecated.